packages/bot-skeleton/src/scratch/blocks/Logic/controls_if.js

```diff
@@ -81,6 +81,7 @@ Blockly.Blocks.controls_if = {
 
         let i = 1;
 
+        /* eslint no-constant-condition: ["error", { "checkLoops": false }]  */
```

Comment:
This will set these rules for the rest of the file, better to use something like:

```JS
// eslint-disable-next-line no-constant-condition
```